pilot/lib/pilot/build_manager.rb

```diff
@@ -218,7 +225,8 @@ def should_update_localized_build_information?(options)
     # If there are fewer than two teams, don't infer the provider.
     def transporter_for_selected_team(options)
       generic_transporter = FastlaneCore::ItunesTransporter.new(options[:username], nil, false, options[:itc_provider])
-      return generic_transporter unless options[:itc_provider].nil? && Spaceship::Tunes.client.teams.count > 1
+      return generic_transporter if options[:itc_provider] || Spaceship::Tunes.client.nil?
```

Comment:
What's this change about?

Reply:
@janpio `Spaceship::Tunes.client.teams.count > 1` would 💣 if not logged in so checking that on the following line separately 😊